src/model/quotes.js

```diff
@@ -73,7 +72,7 @@ class QuotesModel {
     const [payer, payee] = await Promise.all([
       axios.request({ url: `${url}/${headers['fspiop-source']}` }),
       axios.request({ url: `${url}/${headers['fspiop-destination']}` })
-    ].map(p => p.then(res => res.json())))
+    ].map(p => p.then(res => res)))
```

Comment:
Isn't this the same as
```suggestion
    ]
```